src/models/userModel.ts

```diff
@@ -212,6 +226,31 @@ export class UserModel extends Model {
     }
 
     /**
+     * @description Get all the users for a specific company
+     * @param company it's a company id
+     * @returns {Promise<Object>|Promise} Promise that is resolved with user
+     * array or rejected with the error generated from mongoose.
```

Comment:
Non andare a capo in questa maniera. Se vai oltre le 80 colonne usa un paragrafo.
```
<p> Promise that is resolved with user
* array or rejected with the error generated from mongoose. </p>
```